src/pages/AccountSettings/tabs/OrgUploadToken/OrgUploadToken.test.jsx

```diff
@@ -58,73 +58,68 @@ describe('OrgUploadToken', () => {
     }
   ) {
     const user = userEvent.setup()
-    const mutate = jest.fn()
-    const addNotification = jest.fn()
+    const mutate = vi.fn()
+    const addNotification = vi.fn()
 
     useAddNotification.mockReturnValue(addNotification)
 
     server.use(
-      graphql.query('DetailOwner', (req, res, ctx) => {
-        return res(
-          ctx.status(200),
-          ctx.data({
+      graphql.query('DetailOwner', (info) => {
+        return HttpResponse.json({
+          data: {
             owner: {
               ...mockOwner.owner,
               isAdmin: isAdmin,
             },
-          })
-        )
+          },
+        })
       }),
-      graphql.query('GetOrgUploadToken', (req, res, ctx) => {
-        return res(
-          ctx.status(200),
-          ctx.data({
+      graphql.query('GetOrgUploadToken', (info) => {
+        return HttpResponse.json({
+          data: {
             owner: {
               orgUploadToken: orgUploadToken,
             },
-          })
-        )
+          },
+        })
       }),
-      graphql.mutation('regenerateOrgUploadToken', (req, res, ctx) => {
+      graphql.mutation('regenerateOrgUploadToken', (info) => {
         mutate('regenerateOrgUploadToken')
-
-        return res(
-          ctx.status(200),
-          ctx.data({
+        return HttpResponse.json({
+          data: {
             regenerateOrgUploadToken: {
               orgUploadToken,
               error: {
                 __typename: error,
               },
             },
-          })
-        )
+          },
+        })
       })
     )
 
     return { addNotification, mutate, user }
   }
 
   describe('renders component', () => {
-    beforeEach(() => {
```

Comment:
from my own curiosity: what's the thinking behind swapping this out in favor of adding to each test?

Reply:
So imo this gives us better overall control for each individual test, and narrows down potential issues with test setups. It also makes it easier to grab mocks or user events that we setup in the setup functions as well.